packages/amplify_datastore/example/README.md

```diff
@@ -4,13 +4,40 @@ Demonstrates how to use the amplify_datastore plugin.
 
 ## Getting Started
 
-This project is a starting point for a Flutter application.
+### Local mode
 
-A few resources to get you started if this is your first Flutter project:
+This example uses three connected models `Blog`, `Post` and `Comment`. You can run this app in a local only mode with the codegen models already provided in the app. Use the `flutter run` command to run on your connected devices or simulators.
 
-- [Lab: Write your first Flutter app](https://flutter.dev/docs/get-started/codelab)
-- [Cookbook: Useful Flutter samples](https://flutter.dev/docs/cookbook)
+`Post` can only be created for a `Blog`. You fist create a `Blog` and use it in the drop down while creating a `Post`. Similarly `Comment` can be created only for a `Post`.
 
-For help getting started with Flutter, view our
-[online documentation](https://flutter.dev/docs), which offers tutorials,
-samples, guidance on mobile development, and a full API reference.
+Use the `X` on the top to clear your local DataStore
```

Comment:
nit: period at the end of this sentence.

Reply:
Done..